.github/workflows/update-v1docs-gaffer-version.yml

```diff
@@ -0,0 +1,56 @@
+name: Update v1docs Gaffer Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'v1 Gaffer Version'
+        required: true
+
+jobs:
+  update-v1docs-gaffer-version:
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_OPTS: -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25
+      VERSION_UPDATE_BRANCH: updating-v1docs-gaffer-version-${{ github.event.inputs.version }}
+    steps:
+    - name: Checkout v1docs
+      uses: actions/checkout@v2
+      with:
+        ref: v1docs
+        token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+
+    - name: Setup Github credentials
+      run: |
+        git config user.name Gaffer
+        git config user.email github-actions@github.com
```

Comment:
Could we take this opportunity to change this to use the GitHub Actions user, instead of a non existent 'Gaffer'. [As used in the new action](https://github.com/gchq/gaffer-doc/blob/4e6d13d5f48292a3714a854025deac2dfe9592fe/.github/workflows/deploy-docs.yml#L60).

Reply:
Sure thing, I think the `ADMIN_GITHUB_TOKEN` is still needed for ci to run on the created PR though.

Reply:
I don't believe it is required, the other workflow doesn't use it and is performing a write operation. It depends on the repo settings. This is something that might have changed since the action was originally implemented. I don't see any need to change that though, just the user config.